docs/docs-content/clusters/pcg/deploy-app-pcg.md

```diff
@@ -86,27 +89,25 @@ palette login
+| ------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| **Spectro Cloud Console**      | `https://console.spectrocloud.com`. If using a self-hosted instance of Palette, enter the URL for that instance.                                                                                                                                                     | No                       |
+| **Allow Insecure Connection**  | `Y`. Enabling this option bypasses x509 CA verification. In production environments, enter `Y` if you are using a self-hosted Palette or VerteX instance with self-signed TLS certificates and need to provide a file path to the instance CA. Otherwise, enter `N`. | No                       |
+| **Spectro Cloud API Key**      | Enter your Palette API Key.                                                                                                                                                                                                                                          | Yes                      |
+| **Spectro Cloud Organization** | Select your Palette Organization name.                                                                                                                                                                                                                               | Yes                      |
+| **Spectro Cloud Project**      | `None (TenantAdmin)`                                                                                                                                                                                                                                                 | No                       |
+| **Acknowledge**                | Accept the login banner message. [Login banner](../../tenant-settings/login-banner.md) messages are only displayed if the tenant admin enabled a login banner.                                                                                                       | Yes                      |
 
-After accepting the login banner message, you will receive the following output confirming you have successfull
-authentication with Palette.
+After accepting the login banner message, you will receive the following output confirming you have successfully
+authenticated with Palette.
 
 ```text hideClipboard
 Welcome to Spectro Cloud Palette
 ```
 
-The video below demonstrates Palette's authentication process. Ensure you utilize values appropriate for your
-environment.
+The video below demonstrates Palette's authentication process. Ensure you utilize values specific to your environment.
```

Comment:
```suggestion
The video below demonstrates Palette's authentication process. Ensure you utilize values specific to your environment, such as the correct Palette URL. Contact your Palette administrator for the correct URL if you use a self-hosted Palette or VerteX instance. 
```
Thoughts on making the URL part a bit clearer?

Reply:
I liked it :)